src/pages/[platform]/build-a-backend/auth/multi-step-sign-in/index.mdx

```diff
@@ -156,14 +161,58 @@ Future<void> confirmTotpUser(String totpCode) async {
 }
 ```
 
-### Continue signin with MFA Selection
+## Confirm signin with Email MFA
 
-If the next step is `continueSignInWithMFASelection`, the user must select the MFA method to use. Amplify Auth currently only supports SMS and TOTP as MFA methods. After the user selects an MFA method, your implementation must pass the selected MFA method to Amplify Auth using `confirmSignIn` API.
+If the next step is `confirmSignInWithEmailMfaCode`, Amplify Auth has sent the user a random code to their email address and is waiting for the user to verify that code. To handle this step, your app's UI must prompt the user to enter the code. After the user enters the code, pass the value to the `confirmSignIn` API.
+
+<Callout>
+
+The result includes an `AuthCodeDeliveryDetails` member. It includes additional information about the code delivery, such as the partial email address of 
+the recipient, which can be used to prompt the user on where to look for the code.
+
+</Callout>
+
+```dart
+Future<void> _handleSignInResult(SignInResult result) async {
+  switch (result.nextStep.signInStep) {
+    case AuthSignInStep.confirmSignInWithEmailMfaCode:
```

Comment:
'confirmSignInWithEmailMfaCode' has been updated to 'confirmSignInWithOtpCode' in our codebase